### DISCUSSION TITLE
Push notifications

### DISCUSSION THREAD

**Discussion by EnesCeee** [payloadcms/payload#3924] [Q&A]
Hello 👋 guys, I have a mobile project. I need push notifications. I don't see push notifications about in payload cms documants. Can i use push notifications in payload ? Do you know?

**Comment by MarkAtOmniux** [comment]
This could make a cool plugin! I don't see many people needing push notifications built into their CMS, but using a service that allows you to self-host, like ntfy, could be a really cool alternative; https://ntfy.sh/. This could either be a plugin, or it could be something that just runs on the back of a hook :)

**Comment by matthijs166** [comment]
https://onesignal.com/ is also a nice solution to use
